pcdet/datasets/__init__.py

```diff
@@ -12,7 +12,7 @@
 from .pandaset.pandaset_dataset import PandasetDataset
 from .lyft.lyft_dataset import LyftDataset
 from .once.once_dataset import ONCEDataset
-from .argo2.argo2_dataset import Argo2Dataset
+# from .argo2.argo2_dataset import Argo2Dataset
```

Comment:
This change was needed since `.argo2.argo2_dataset` requires the argo toolkit to be installed. I think it's just a bug in that file, but this was the easiest fix.